packages/player/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+	"extends": "../tsconfig.settings.json",
+	"compilerOptions": {
+		"composite": true,
+		"rootDir": "./src",
+		"outDir": "./dist",
+		"skipLibCheck": true,
+		"jsx": "react-jsx"
+	},
+	"references": [
+		{"path": "../core"},
+		{"path": "../bundler"},
+		{"path": "../renderer"}
```

Comment:
May remove bundle and renderer from references, neither should be accessed by renderer.

Reply:
Okay Noted.